currency-service/prometheus.yml

```diff
@@ -6,5 +6,9 @@ scrape_configs:
     metrics_path: '/actuator/prometheus'
     static_configs:
       - targets: ['192.168.0.73:8080'] # use your IP -> TODO generic value
+  - job_name: 'configuration-server'
+    metrics_path: '/actuator/prometheus'
+    static_configs:
+      - targets: ['192.168.0.73:8071'] # use your IP -> TODO generic value
```

Comment:
sowohl im docker-compose als auch in kubernetes gibts es einen internen dns. dh, wir könnten dann hier die namen der container verwenden.

Reply:
Habe es letzten Freitag probiert, habe hatte keine Zeit mehr. Ist eher gerade quick and dirty bis alle container fertig sind.